src/webgl/p5.RendererGL.Retained.js

```diff
@@ -75,7 +75,12 @@ p5.RendererGL.prototype.createBuffers = function(gId, model) {
     // allocate space for faces
     if (!indexBuffer) indexBuffer = buffers.indexBuffer = gl.createBuffer();
     const vals = p5.RendererGL.prototype._flatten(model.faces);
-    this._bindBuffer(indexBuffer, gl.ELEMENT_ARRAY_BUFFER, vals, Uint16Array);
+    const maxVal = vals.reduce((v, m) => (v > m ? v : m), 0);
```

Comment:
Thanks for this! My only request is for a short comment to explain the need for this.

Reply:
I've added some comments and I also realized that I didn't need to actually reduce the entire face list to find the maximum vertex index, I just needed to check if any vertex index is over max uint16.